index.js

```diff
@@ -224,6 +236,38 @@ const fillFieldWithTextWhenVisibleAndPressEnter = function(
   );
 };
 
+const scrollToElement = function(htmlElement)
```

Comment:
I imagine this was created only because it is used by the other new methods. What about defined it as you defined the `waitForElementToBeClickable`, and not export it?